src/components/DateInput/DateInput.tsx

```diff
@@ -46,6 +51,7 @@ const DateInput: React.FC<DateInputProps & Omit<TextInputProps, 'value' | 'onCha
   value,
   format = 'MM/DD/YYYY',
   withTime,
+  mode = '12h',
```

Comment:
Perhaps default should be "24h"?

In Panther I don't think we'll ever use  `12h`, but I think that you made the right choice to have it as an option. I'd simply change the default value to be the one we'd be using in Panther (protecting us in case any dev forgets  to set it)

Reply:
Done!